### PR TITLE
feat: web interface, scan-stats and jellyfin and autoscan targets

### PR DESCRIPTION
**Pull request by l3uddz**
**Fixes**

- Processor should not run when there are no targets configured (caused scans to be removed)
- Do not display no authentication warning log when a lidarr trigger has been setup
- Potentially fixes bernard sync data anomaly issues

**New**

- Submit scans requests via a web form at `/triggers/manual`
- Autoscan is now a target.
- Jellyfin is now a target.
- New build system resulting in multiarch binaries and docker images.
- Added `scan-stats` config option to display remaining scans on a regular basis (defaults to 60 minutes)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,11 +27,9 @@ Alternatively, you can build the Autoscan binary yourself.
 To build the autoscan CLI on your system, make sure:
 
 1. Your machine runs Linux, macOS or WSL2
-2. You have [Go](https://golang.org/doc/install) installed (1.14 or later preferred)
-3. You have a GCC compiler present \
-  *Yup, we need to link to C because of SQLite >:(*
-4. Clone this repository and cd into it from the terminal
-5. Run `go build -o autoscan ./cmd/autoscan` from the terminal
+2. You have [Go](https://golang.org/doc/install) installed (1.16 or later preferred)
```
Comment (m-rots) [inline README.md:30]: Go embed requires 1.16 or later, so it's required instead of preferred.

**Review by m-rots — CHANGES_REQUESTED** [review]
Skipped the `template.html`, might look into it later.

**Review by m-rots — COMMENTED** [review]
Looking good! Waiting for the other changes

File: cmd/autoscan/main.go [hunk 2]
```diff
@@ -322,17 +351,69 @@ func main() {
...
+
 	// processor
 	log.Info().Msg("Processor started")
 
 	targetsAvailable := false
-
+	targetsSize := len(targets)
 	for {
+		// sleep indefinitely when no targets setup
+		if targetsSize == 0 {
```
[the remark this answers is not in the document]
Comment (l3uddz) [reply cmd/autoscan/main.go:388]: i think having the variable is better than constant size checking every 15 seconds (even if the hit is minimal)

**Review by m-rots — APPROVED** [review]
AUTOSCAN 1.2 HYPE TRAIN 🚀🚀🚀

**Pull request merged**